### DISCUSSION TITLE
Can I use AWS CodeCommit as a git repo?

### DISCUSSION THREAD

**Discussion by nikolaydyankov** [vercel/community#3284] [Feature Question] [answered]
### Question

Hi, I'm trying to add my CodeCommit repo as a private git repo, but I'm getting "Invalid repository URL.". I'm pasting the full URL:

`https://git-codecommit.eu-central-1.amazonaws.com/v1/repos/<repo>`

Is CodeCommit not supported?

Thanks

**Comment by swarnava** [comment, marked as answer]
Hi,

Code Commit isn't supported via native integration unfortunately. You need to use custom workflow and deploy using Vercel CLI.

**Reply by nikolaydyankov** [reply]
I haven't looked into Vercel in detail, is that possible with the Hobby plan?

**Reply by swarnava** [reply]
Hi, yes it is possible on Hobby plan.

**Reply by nikolaydyankov** [reply]
Alright, thanks!
